### DISCUSSION TITLE
Error in css file after build

### DISCUSSION THREAD

**Discussion by Dev-Phack** [tailwindlabs/tailwindcss#7910] [Help] [answered]
When trying to install tailwindcss with the cli I get an error in the generated css code after build. so I can't use any class from tailwindcss. I'm not using a framework, I'm installing in a completely empty project. Can anyone help me?
The error occurs from line 732 in the built css file. The problematic code is as follows:

`.w-\[this-is\\\\\]w-\\\\\[weird-but-valid\] {
  width: this-is\\]w-\\[weird-but-valid;
}
`

**Comment by adamwathan** [comment, marked as answer]
See previous discussion that explains the issue here: https://github.com/tailwindlabs/tailwindcss/discussions/5781

**Reply by Dev-Phack** [reply]
Okay, thanks. It's solved!
